source/_patterns/20-molecules/teaser/teaser.scss

```diff
@@ -70,7 +70,7 @@
     }
   }
 
-  // if any Teaser(size) is beeing clicked … do the following with it's siblings and children
+  // if any Teaser(size) is being clicked … do the following with it's siblings and children
```

Comment:
its = possessive pronoun (a duck and its children)
it's = short for `it is` (it's a plane)

http://www.its-not-its.info/

Reply:
HAH, ja eigentlich logisch :D